textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/common/JoinPredicate.java

```diff
@@ -0,0 +1,59 @@
+package edu.uci.ics.textdb.dataflow.common;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IPredicate;
+
+/**
+ * 
+ * @author sripadks
+ *
+ */
+public class JoinPredicate implements IPredicate {
+	
+	private Attribute outerIdAttribute;
+	private Attribute outerJoinAttribute;
+	private Attribute innerIdAttribute;
+	private Attribute innerJoinAttribute;
+	private Integer threshold;
+	
+	/**
+	 * This constructor is used to set the parameters required for the Join Operator.
+	 * 
+	 * JoinPredicate joinPre = new JoinPredicate(ID_ATTR, DESCRIPTION_ATTR, ID_ATTR, DESCRIPTION_ATTR, 10)
+	 * will create a predicate that compares the spans of type DESCRIPTION_ATTR of outer and inner operator
```

Comment:
Add a real example with two tuples including spans to show the meaning of this predicate.